pkg/products/threescale/reconciler.go

```diff
@@ -390,30 +390,46 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
-	ingressRouterService, err := customDomain.GetIngressRouterService(ctx, serverClient)
-	if err != nil || len(ingressRouterService.Status.LoadBalancer.Ingress) == 0 {
-		errorMessage := "failed to retrieve ingress router service"
-		r.log.Error(errorMessage, err)
-		phase = integreatlyv1alpha1.PhaseFailed
-		events.HandleError(r.recorder, installation, phase, errorMessage, err)
-		return phase, fmt.Errorf("%s: %v", errorMessage, err)
+	threescaleMasterRoute, err := r.getThreescaleRoute(ctx, serverClient, "system-master", nil)
+	if err != nil || threescaleMasterRoute == nil {
+		return integreatlyv1alpha1.PhaseFailed, err
 	}
 
-	ips, err := customDomain.GetIngressRouterIPs(ingressRouterService.Status.LoadBalancer.Ingress)
-	if err != nil {
-		errorMessage := "failed to retrieve ingress router ips"
-		r.log.Error(errorMessage, err)
-		phase = integreatlyv1alpha1.PhaseFailed
-		events.HandleError(r.recorder, installation, phase, errorMessage, err)
-		return phase, fmt.Errorf("%s: %v", errorMessage, err)
+	listOfIngresServicesNames := []string{"router-default"}
+	if customDomainActive {
```

Comment:
We should not only do this if custom domain active. There maybe another ingress router even if this is false. Probably unlikely but we should check anyway

Reply:
Logic was changed and moved ping3scalePortals